changelogs/fragments/2246-terraform.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - terraform - fix issue that cause the execution fail because from Terraform 0.15 ``-var`` and ``-var-file`` options are no longer available on terraform validate (https://github.com/ansible-collections/community.general/pull/2246).
+  - terraform - fix issue that cause the destroy to fail because from Terraform 0.15 ``terraform destroy -force`` option is replaced with ``terraform destry -auto-approve`` (https://github.com/ansible-collections/community.general/issues/2247).
```

Comment:
```suggestion
  - terraform - fix issue that cause the execution fail because from Terraform 0.15 on, the ``-var`` and ``-var-file`` options are no longer available on ``terraform validate`` (https://github.com/ansible-collections/community.general/pull/2246).
  - terraform - fix issue that cause the destroy to fail because from Terraform 0.15 on, the ``terraform destroy -force`` option is replaced with ``terraform destroy -auto-approve`` (https://github.com/ansible-collections/community.general/issues/2247).
```

Reply:
What about this?